package.json

```diff
@@ -84,8 +84,9 @@
     "@types/react": "^17.0.69",
     "@types/react-dom": "^17.0.22",
     "@types/svgo": "^2.6.4",
-    "@typescript-eslint/eslint-plugin": "^5.59.8",
-    "@typescript-eslint/parser": "^5.59.8",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
```

Comment:
These updates were necessary in order to use v8 compatible types.  This doesn't mean that only v8 and up are support, just that internally, we need to have tall of the type information for v8 context objects to fully support v8 (and 9) in our TS-based rules.